tests/components/zwave_js/conftest.py

```diff
@@ -82,6 +82,12 @@ def nortek_thermostat_state_fixture():
     return json.loads(load_fixture("zwave_js/nortek_thermostat_state.json"))
 
 
+@pytest.fixture(name="window_cover_state", scope="session")
+def window_cover_state_fixture():
+    """Load the window cover node state fixture data."""
+    return json.loads(load_fixture("zwave_js/window_cover_state.json"))
```

Comment:
We should try to name the fixture after the device name and not the type of device. Then we know more specifically what device it is.

Reply:
That's the plan, unfortunately the dump wasn't complete.

Reply:
```suggestion
    return json.loads(load_fixture("zwave_js/chain_actuator_zws12_state.json"))
```

Reply:
Is leaving the fixture name generic ok? "window_cover" ?

Reply:
Please use my suggestion and rename the file.